CITATION.cff

```diff
@@ -0,0 +1,14 @@
+cff-version: 1.2.0
+message: "If Iris played an important part in your research then please add us to your reference list by using the references below."
+authors:
+    - name: "Met Office"
+      address: "Exeter, Devon"
```

Comment:
I kinda disagree with this.

I'd prefer to see individual entries from those that have contributed to `iris`, so that's everyone in the community, not just those at the Met Office as an institution.

i.e., an entry such as
```
- family-names: "Little"
  given-names: "Bill"
  orcid: "https://orcid.org/0000-0002-1345-9465"
```
So those who want to be cited as an author of their contribution can be explicitly using [ORCID](https://orcid.org/).

This appears to be at the approach taken by other open source projects.